src/meta/metainfo.go

```diff
@@ -353,6 +354,8 @@ type PropertyInfo struct {
 	Pos         ElementPosition
 	Typ         TypesMap
 	AccessLevel AccessLevel
+	FromDoc     bool
+	IsVerified  bool
```

Comment:
`IsVerified` looks like a strange name if taken in comparison with `FromDoc`. It looks like it's used as `FromDecl` when the property is defined using a PHP declaration instead of `@property`.

Why do we have IsVerified only for PropertyInfo, but not for the FuncInfo?

Reply:
I could not come up with a good name, but this flag is needed so that there are no false positives.

This flag is set to true only when this property has been added to functions where duplicates are checked (`enterPropertyList`) if the property in the class has already been added from somewhere outside (because there have been cases when the class already had some properties (not from documentation), but class processing has just begun), then this flag will always be false, thereby only those properties that were added to `enterPropertyList` are checked, which will help to avoid any false positives.

We can say that with this flag, the `FromDoc` flag becomes redundant, since for all properties from the documentation `IsVerified` will be false.

Reply:
Probably the truth is that can rename this property `FromDecl` and leave only it.